### DISCUSSION TITLE
分账的功能怎么使用？

### DISCUSSION THREAD

**Discussion by emyshen** [yansongda/pay#980] [General]
插件里面的分账功能该怎么调用？V3版本的

**Comment by yansongda** [comment]
1、使用 shortcut

参考 https://github.com/yansongda/pay/tree/master/src/Shortcut/Wechat 对 https://github.com/yansongda/pay/tree/master/src/Plugin/Wechat/V3/Extend/ProfitSharing 里面需要的功能进行 shortcut

2、直接使用对应插件

参考 https://pay.yansongda.cn/docs/v3/kernel/kernel.html 使用 https://github.com/yansongda/pay/tree/master/src/Plugin/Wechat/V3/Extend/ProfitSharing 里的相关插件
